app/frontend/src/components/HorizontalScroller.tsx

```diff
@@ -8,23 +8,29 @@ import useOnVisibleEffect from "../utils/useOnVisibleEffect";
 import CircularProgress from "./CircularProgress";
 
 const useStyles = makeStyles((theme) => ({
-  padder: {
-    height: 1,
-    width: 0.1,
-  },
   root: {
     "& > *": {
-      flex: "0 0 auto",
-    },
-    "& > * + *": {
-      marginInlineStart: theme.spacing(2),
+      flexShrink: 0,
+      height: "calc(100% - 2rem)",
+      width: "85vw",
+      maxWidth: "33rem",
```

Comment:
I thought it's better to let the children decide their width

Reply:
Without a set width, the card would grow as large as the text inside.

Reply:
This component takes `children` and the children can set their width. For example, the events in the community page fit a few in one screen, even on mobile, but now there is only one fully visible